<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenCombate</class>
 <widget class="QDialog" name="ScreenCombate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="fusion1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>FUSIÓN</string>
   </property>
  </widget>
  <widget class="QLabel" name="mensaje1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI Semilight</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="Salir">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>560</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxu1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>530</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="barrapoder1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
   <property name="format">
    <string>%v/10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cambiar1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>CAMBIAR</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="barravida1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="format">
    <string>%v/100</string>
   </property>
  </widget>
  <widget class="QLabel" name="picpersonaje1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>331</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="usar1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>USAR</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxf1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="barrapoder2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
   <property name="format">
    <string>%v/10</string>
   </property>
  </widget>
  <widget class="QLabel" name="picpersonaje2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>70</y>
     <width>301</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="fusion2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>FUSIÓN</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxu2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>530</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxf2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>530</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="barravida2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>10</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="format">
    <string>%v/100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cambiar2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>CAMBIAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="usar2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>USAR</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxc1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxc2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>530</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="picfondo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>-10</y>
     <width>1014</width>
     <height>630</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>picfondo</zorder>
  <zorder>fusion1</zorder>
  <zorder>mensaje1</zorder>
  <zorder>Salir</zorder>
  <zorder>comboBoxu1</zorder>
  <zorder>barrapoder1</zorder>
  <zorder>cambiar1</zorder>
  <zorder>barravida1</zorder>
  <zorder>picpersonaje1</zorder>
  <zorder>usar1</zorder>
  <zorder>comboBoxf1</zorder>
  <zorder>barrapoder2</zorder>
  <zorder>picpersonaje2</zorder>
  <zorder>fusion2</zorder>
  <zorder>comboBoxu2</zorder>
  <zorder>comboBoxf2</zorder>
  <zorder>barravida2</zorder>
  <zorder>cambiar2</zorder>
  <zorder>usar2</zorder>
  <zorder>comboBoxc1</zorder>
  <zorder>comboBoxc2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
